pkg/pillar/cpuallocator/cpuallocator.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2022 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package cpuallocator
+
+import (
+	"fmt"
+	"sync"
+)
```

Comment:
I don't know why golangcilint complains here, but normally we have a single import statement, with a blank line between the standard imports and the github imports.

Reply:
Should I fix something here?

Reply:
Fixed